commandlineparser/include/Parser.h

```diff
@@ -0,0 +1,23 @@
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <sstream>
+#include <map>
+
+#include "Options.h"
+
+#ifndef PARSER_H
+#define PARSER_H
+
+namespace cmdParser
+{
+	class Parser 
+	{
+		std::map<std::string ,opts::Options*> command_list; 
```

Comment:
please create a alias for map declaretype 

using commandList = std::map<std::string ,opts::Options*>

and then you can use commandList  instead std::map<std::string ,opts::Options*>